### DISCUSSION TITLE
Please add a link to my profile on the homepage!

### DISCUSSION THREAD

**Discussion by loeclos** [community/community#125088] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

 Please add a link to my profile on the homepage! What I have to do right now is to press onto one of my repos and then navigate to my my profile through the link at the top:

 
![image](https://github.com/community/community/assets/116607327/76f7146a-eba1-40c2-b668-c5747ca1ff00)

I think that it would be a lot easier just to add a direct link to the user's profile   on the homepage.

**Comment by ebndev** [comment]
Hi @marsianjohncarter,

To get to your profile you can click on your photo in the top right corner and see a list of all of different things you can access on GitHub. Hope that helps!

<img width="248" alt="Screenshot 2024-05-22 at 12 41 49 PM" src="https://github.com/community/community/assets/164093511/bdb7299f-db70-49d8-99f6-77bc1349ab87">

**Reply by loeclos** [reply]
Hey @ebndev thanks for the tip!

**Comment by aryanjsx** [comment]
![image](https://github.com/community/community/assets/57598770/015f335d-2dc7-41ac-a70e-4312aa09856c)
You can simply click on the the github icon. to visit on the home page
